news/2019-07-07-beatmap-spotlights-spring-2019.md

```diff
@@ -0,0 +1,302 @@
+
+## Winter Spotlights results
+
+First of all, congratulations to the winners of the Seasonal Spotlights: Winter 2019. They will receive 3, 2, and 1 month of supporter for 1st, 2nd, and 3rd place respectively for their efforts!
+
+- **osu!:** [Toy](https://osu.ppy.sh/users/2757689), [Apraxia](https://osu.ppy.sh/users/4194445) and [xxluizxx47](https://osu.ppy.sh/users/4687701)
+- **osu!taiko:** [Jaye](https://osu.ppy.sh/users/4841352), [AlvaroYL](https://osu.ppy.sh/users/6333166) and [tasuke912](https://osu.ppy.sh/users/2774767)
+- **osu!catch:** [rostld](https://osu.ppy.sh/users/4365562), [RAMPAGE88](https://osu.ppy.sh/users/448547) and [-Nirfu-](https://osu.ppy.sh/users/7332050)
+- **osu!mania:** [\[Crz\]Satori](https://osu.ppy.sh/users/7082178), [Stink God](https://osu.ppy.sh/users/7381289) and [Cryolien](https://osu.ppy.sh/users/1626983)
+
+
+# Navigation
+
+- <a href="#osu">osu!</a>
+- <a href="#taiko">osu!taiko</a>
+- <a href="#catch">osu!catch</a>
+- <a href="#mania">osu!mania</a>
+
+# Spring 2019 Spotlights
+
+## <a name="osu" id="osu"></a>osu!
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/osu/kagefumi.jpg)](https://osu.ppy.sh/beatmapsets/910454)
+
+A mapper of repute in the community for their unorthodox music taste and style, [squirrelpascals](https://osu.ppy.sh/users/6151332) brings forth a very fitting interpretation of Hyakkei's "kagefumi". From unique stream patterns to minimalistic slider velocities to characterized slider shapes and much more, the mapper underlines the core portions of the chill, tranquil math rock song with clarity, and accentuates their features fantastically with distinctive design choices that keep with the song's simplicity.
+
+Those unprepared for the challenge may find a very engaging experience as well in the insane difficulty, although skilled players would not wish to sleep on this one too. With more emphasis on greater slider movements, the difficulties compliment one another with their slight varied approaches to the song, while both capturing its integrity faithfully. A truly refreshing mapset overall that sits distinguished among the many maps in the ranked section!
+
+_written by [faygo](https://osu.ppy.sh/users/5199686)
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/osu/uh.jpg)](https://osu.ppy.sh/beatmapsets/948199)
+
+[sammish](https://osu.ppy.sh/users/4415076) manages to make 17 minutes feel like 5 in his map of Tennyson's "Uh Oh! EP". The hard work he puts into expressing each song's personality shows- and obviously pays off. The map feels smooth to play all the way through and aligns very closely to Tennyson's unique, funky style of music. This is a phenomenal debut of both sammish and Tennyson in the ranked section and we hope to see more from both in the future!
```

Comment:
```suggestion
[sammish](https://osu.ppy.sh/users/4415076) manages to make 17 minutes feel like 5 in his map of Tennyson's "Uh Oh! EP". The hard work he puts into expressing each song's personality shows—and obviously pays off. The map feels smooth to play all the way through and aligns very closely to Tennyson's unique, funky style of music. This is a phenomenal debut of both sammish and Tennyson in the ranked section and we hope to see more from both in the future!
```

Reply:
```suggestion
[sammish](https://osu.ppy.sh/users/4415076) manages to make 17 minutes feel like 5 in his map of Tennyson's "Uh Oh! EP". The hard work he puts into expressing each song's personality shows — and obviously pays off. The map feels smooth to play all the way through and aligns very closely to Tennyson's unique, funky style of music. This is a phenomenal debut of both sammish and Tennyson in the ranked section and we hope to see more from both in the future!
```

Reply:
https://www.thepunctuationguide.com/em-dash.html  See the end of this article where it talks about the em dash used in newspapers etc.  Typically they have a space before and after to help with readability.  This also follows the very common "AP style" format